src/Salt/UserBundle/Listener/LoginListener.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Salt\UserBundle\Listener;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
+use Symfony\Component\Security\Core\Exception\BadCredentialsException;
+use Symfony\Component\Security\Http\Event\InteractiveLoginEvent;
+use ReCaptcha\ReCaptcha;
+
+class LoginListener
+{
+    private $container;
+
+    public function __construct(Container $container)
+    {
+        $this->container = $container;
+    }
+
+    public function onSecurityInteractiveLogin(InteractiveLoginEvent $event)
+    {
+        $request = $event->getRequest();
+        $captcha_secret = $this->container->getParameter('google_captcha_secret_key');
```

Comment:
Would be better to just pass the parameter into the constructor instead of the entire container.